example/example-node-headings.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable no-console */
```

Comment:
I guess we can have the headings added to the existing example-node.js and example.js at the same place, and avoid this new example file.